.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
+/.phpunit
```

Comment:
Certainly not, see https://github.com/symfony/symfony/issues/13934#issuecomment-131820307